apps/codeimage/src/components/Pagination/Pagination.tsx

```diff
@@ -20,17 +20,28 @@ const Pagination = (props: ParentProps<PaginationProps>) => {
     page => props.onChange(page),
     false,
   );
+
+  const handleChange = (direction: 'next' | 'prev') => () =>
+    merged.onChange(prev => (direction === 'next' ? prev + 1 : prev - 1));
+
```

Comment:
@riccardoperra do you know why in this clousure eslint suggest me to do it in createEffect,? maybe I don't do this?

Reply:
handle-change is already a callback. Also, it should be separated into two function `increment` and `decrement` instead of one